common/test/acceptance/tests/lms/test_lms_problems.py

```diff
@@ -459,9 +459,29 @@ def test_question_with_description(self):
         self.assertEqual(problem_page.problem_question_descriptions, self.descriptions)
 
 
+class CAPAProblemA11yBaseTestMixin(object):
+    """Base TestCase Class to verify CAPA problem accessibility."""
+
+    def test_a11y(self):
+        """
+        Verifies that there are no accessibility issues for a particular problem type
+        """
+        self.courseware_page.visit()
+        problem_page = ProblemPage(self.browser)
+
+        # Set the scope to the problem question
+        problem_page.a11y_audit.config.set_scope(
+            include=['section.wrapper-problem-response']
+        )
+
+        # Run the accessibility audit.
+        problem_page.a11y_audit.check_for_accessibility_errors()
+
+
 @attr('a11y')
-class CAPAProblemQuestionDescriptionA11yTest(ProblemsTest):
+class CAPAProblemQuestionDescriptionA11yTest(CAPAProblemA11yBaseTestMixin, ProblemsTest):
```

Comment:
I think we should convert this class to multiple choice or checkboxes rather than generic one.